lib/WebpackError.js

```diff
@@ -0,0 +1,21 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Jarid Margolin @jaridmargolin
+*/
+"use strict";
+
+module.exports = class WebpackError extends Error {
+	constructor() {
+		super();
+
+		if(this.initialize) {
+			this.initialize.apply(this, arguments);
```

Comment:
It's a bad practice to call members in the constructor.

Could you leave the constructors untouched and only add the base class with the inspect() method to the errors?